applications/ultrasound_postprocessing/tests/test_pipeline_core.py

```
@@ -17,22 +17,23 @@
-    run_pipeline,
-)
-from ultra_post.filters.registry import FILTERS
+    from ultra_post.core.pipeline import (
+        Pipeline,
+        create_node,
+        pipeline_from_yaml,
+        pipeline_to_yaml,
+        run_pipeline,
+    )
+    from ultra_post.filters.registry import FILTERS
+except Exception as exc:  # pragma: no cover - optional GPU dependency
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)